src/AddTodoForm.js

```diff
@@ -1,11 +1,19 @@
 import react from "react";
 function AddTodoForm() {
+    const handleAddTodo = (event) => {
+        event.preventDefault();
+        setNewTodo(event.target.value);
```

Comment:
After we have set `props` as an argument above, we can then use `props.setNewTodo` here.